integration_tests/snaps/kernel-download/snapcraft.yaml

```diff
@@ -0,0 +1,10 @@
+name: kernel-download
+version: 0.1
+summary: Simple kernel snap to test download
+description: test
+type: kernel
```

Comment:
you don't really need this to be a kernel snap, you just need to use the kernel plugin ;-)